script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -409,10 +409,10 @@ The cold but warm feeling left me.
 <Page114>
 "But Sensei. I feel uneasy when I see these lines. If I touch a line, it will break, right? Then it wouldn't be strange if I cut something around me by accident."
 <Page115>
-"I'll take care of that problem.
+"I'll somehow take care of that problem.
```

Comment:
Somehow would be more natural at the end of the sentence ("I'll take care of that problem somehow.")